SPECS/python-packaging/python-packaging.spec

```diff
@@ -55,6 +55,9 @@ Core utilities for Python packages
 %license LICENSE
 
 %changelog
+* Thu Aug 29 2024 Andrew Phelps <anphel@microsoft.com> - 23.2-3
+- Bump release to rebuild and resolve python3dist provides issue
```

Comment:
I think we should consider removing explicit `Provides` from `python-setuptools` and rebuilding it as well. This way we won't miss any provides, which may appear after we upgrade that package.

Reply:
I've rebuilt python-setuptools, although it still does not appear to get the automatic provides. So I've left the hard-coding for now.

Reply:
That one sounds worrying but we can investigate it separately. Could you create a bug for it, though? Fedora's spec doesn't have these particular explicit `Provides` but their RPM still lists them, so it looks like we might be missing something.

Reply:
Sure, I've filed this tracking bug: https://microsoft.visualstudio.com/OS/_workitems/edit/53646342